src/app/components/core/system-alert/system-alert.component.ts

```diff
@@ -35,8 +42,11 @@ export class SystemAlertComponent {
   getAlerts() {
     this.systemAlertService.getAlerts().
     subscribe(
-      (data) => {
-        this.systemAlerts = data['alerts'];
+      (data) => {   
```

Comment:
Typecast  `data: SystemAlerts[]`?

Reply:
Also, cannot We access it as `this.systemAlerts = data.alerts;`